src/interactions/focusable.js

```diff
@@ -25,6 +25,7 @@ import { find } from './find';
 export function focus(selector) {
   return find.call(this, selector)
     .do(($node) => {
+      $node.focus();
```

Comment:
Minor detail, can we get a newline after this since the next statement is multi-line?